ziggurat-core-crawler/Cargo.toml

```diff
@@ -9,5 +9,5 @@ repository = "https://github.com/runziggurat/ziggurat-core"
 description = "A crawler package for ziggurat-based projects"
 
 [dependencies]
-serde = {version = "1", features = ["derive"]}
-spectre = { git = "https://github.com/niklaslong/spectre", rev = "61bdbf5" }
\ No newline at end of file
+serde = { version = "1", features = ["derive"] }
```

Comment:
maybe we should now use Niklas' latest `crates.io` release, "0.5.1"

Reply:
I've looked through the changes and updated dependency.